### PR TITLE
doc: describe mark_events logging available via the OSD's OpTracker

### PR DESCRIPTION
**Pull request by gregsfortytwo**
Signed-off-by: Greg Farnum <gfarnum@redhat.com>

### CODE REVIEW

File: doc/rados/troubleshooting/troubleshooting-osd.rst [hunk 1]
```diff
@@ -417,7 +417,53 @@ Possible solutions
...
 
-
+Debugging Slow Requests
+-----------------------
+
+If you dump_historic_ops or dump_ops_in_flight, you will see a set of
+operations and a list of events each operation went through. These are briefly
+described below.
+
+Events from the Messenger layer:
```
Comment (jdurgin) [inline doc/rados/troubleshooting/troubleshooting-osd.rst]: for rst to format the list you'll need a blank line after each header

File: doc/rados/troubleshooting/troubleshooting-osd.rst [hunk 2]
```diff
@@ -417,7 +417,53 @@ Possible solutions
 - Upgrade Ceph
 - Restart OSDs
 
-
+Debugging Slow Requests
+-----------------------
+
+If you dump_historic_ops or dump_ops_in_flight, you will see a set of
```
Comment (jdurgin) [inline doc/rados/troubleshooting/troubleshooting-osd.rst]: mention the full command - not clear these are osd asok commands

**Comment by jdurgin** [comment]
submodules struck again

**Comment by gregsfortytwo** [comment]
Fixed @jdurgin.

**Pull request merged**